src/pages/admin/EventEdit.js

```diff
@@ -29,18 +33,22 @@ const useStyles = makeStyles(theme => ({
 
 function EventEdit(props) {
     const classes = useStyles();
-
     const { id: eventId } = useParams();
     const [event, setEvent] = useState(null);
+    const [previewEvent, setPreviewEvent] = useState({});
+    const history = useHistory();
 
     const { events } = props;
+    if (!events) {
+        updateEvents()
+    }
 
-    useEffect(() => {
-        // Get the initial values
-        if (!event && events && eventId) {
-            setEvent(events.find(event => event.id === eventId));
-        }
-    }, [event, events, setEvent, eventId])
+    // Get the initial values
+    if (!event && events && eventId) {
+        let event = events.find(event => event.id === eventId)
```

Comment:
would it make sense to set this using `const` instead of `let`

Reply:
yea